content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -32,6 +32,9 @@ If you need to upgrade Jenkins as well as the JVM, we recommend that you:
 If you use Docker containers to run Jenkins, the default Docker containers use Java 11 beginning with link:/doc/upgrade-guide/2.303/#upgrading-to-jenkins-lts-2-303-1[Jenkins 2.303.1].
 If you need to remain with Java 8 in a Docker container, append `-jdk8` to the Docker image tag.
 
+Starting with the 2.353 weekly release line, Java 11 will be required for future weekly release lines.
+These instructions will still apply to the Long Term Support release line, until the LTS release line mirrors the weekly release line Java requirements.
```

Comment:
This sounds an awful lot like "it will until it won't", raising or implying the question (to me) "when or why won't it"? I suppose the "why" should be obvious to anyone who understands the concept of weeklies and LTS, but the "when" doesn't seem obvious to me, so I think we should include that.

```suggestion
These instructions will still apply to the Long Term Support release line until September 2022, when the LTS release line will mirror the weekly release line Java requirements.
```

Reply:
Thank you for the suggestion @basil, I definitely understand what you're saying and agree that it would be much more clear with that information added.